components/member-list-item/index.js

```diff
@@ -19,25 +19,35 @@ const PreviewMember = (props) => {
   }, []);
 
   return (
-    <Link prefetch={false} href="/members/[id]" as={`/members/${rdsId}`} key={rdsId}>
-      <div className={classNames.container}>
-        <div>
+    <div className={classNames.container}>
+      <Link
+        prefetch={false}
+        href={{
+          pathname: '/members/[id]',
+          query: {
+            first_name: `${data ? data.first_name : ''}`,
+            last_name: `${data ? data.last_name : ''}`
+          }
+        }}
+        as={`/members/${rdsId}`}
+        key={rdsId}>
+        <a href={`/members/${rdsId}`}>
           <div ref={imgRef} className={classNames.imgContainer}></div>
-        </div>
+        </a>
+      </Link>
 
-        <h2 className={classNames.nameOfPerson}>
-          {data ? `${data['first_name']} ${data['last_name']}` : rdsId}
-        </h2>
-        {data && (
-          <div className={classNames.iconsContainer}>
-            {data['twitter_id'] && <SocialMediaIcon id={data.twitter_id} type="twitter" />}
-            {data['github_id'] && <SocialMediaIcon id={data.github_id} type="github" />}
-            {data['linkedin_id'] && <SocialMediaIcon id={data.linkedin_id} type="linkedIn" />}
-            {data['instagram_id'] && <SocialMediaIcon id={data.instagram_id} type="instagram" />}
-          </div>
-        )}
-      </div>
-    </Link>
+      <h2 className={classNames.nameOfPerson}>
+        {data ? `${data['first_name']} ${data['last_name']}` : rdsId}
+      </h2>
+      {data && (
+        <div className={classNames.iconsContainer}>
+          {data['twitter_id'] && <SocialMediaIcon id={data.twitter_id} type="twitter" />}
+          {data['github_id'] && <SocialMediaIcon id={data.github_id} type="github" />}
+          {data['linkedin_id'] && <SocialMediaIcon id={data.linkedin_id} type="linkedIn" />}
+          {data['instagram_id'] && <SocialMediaIcon id={data.instagram_id} type="instagram" />}
```

Comment:
NIT: Let's move the "complex" variable conditionals above the `return`

e.g
```js
const { twitter_id: twitterID } = data;

return {
 { twitterID && <SocialMediaIcon id={twitterID} /> } 
}
```

Reply:
Can we have another issue/PR for this as this is not related to Page Title?

Reply:
I can see that this code was touched in this PR @whyDontI

Reply:
I think I have just moved it around I guess.
@tejasberi has worked on it.
![Screenshot from 2020-08-12 11-54-57](https://user-images.githubusercontent.com/26569942/89982158-d085ff80-dc92-11ea-8751-4715172d9160.png)

Reply:
whats the issue here?